api/types/server.go

```diff
@@ -42,8 +42,6 @@ type Server interface {
 	GetHostname() string
 	// GetNamespace returns server namespace
 	GetNamespace() string
-	// GetLabels returns server's static label key pairs
-	GetLabels() map[string]string
```

Comment:
Technically, this is a backwards incompatible change in the API module interface which people may be relying on. I would keep the original naming.